lib/entity/k_line_entity.dart

```diff
@@ -1,45 +1,45 @@
 import '../entity/k_entity.dart';
 
 class KLineEntity extends KEntity {
-  double open;
-  double high;
-  double low;
-  double close;
-  double vol;
-  double amount;
-  double change;
-  double ratio;
-  int time;
+  late double open;
+  late double high;
+  late double low;
+  late double close;
+  late double vol;
+  late double amount;
+  double? change;
+  double? ratio;
+  int? time;
 
   KLineEntity.fromCustom({
-    this.amount,
-    this.open,
-    this.close,
+    required this.amount,
+    required this.open,
+    required this.close,
     this.change,
     this.ratio,
-    this.time,
-    this.high,
-    this.low,
-    this.vol,
+    required this.time,
```

Comment:
Because of the non-standard data, `time==null` was added.
Please consider the compatibility of the example.
```
if (time == null) {      time = (json['id']?.toInt());      if (time != null) {        time = time! * 1000;      }    }
```
Such as use `int? tempTime` in function.

Reply:
what happend if `json['id']?.toInt()` is `null`

Reply:
in example data, this will not happen. So just use `!` when id is null.

Reply:
A better way is solving `time` in the demo.

Reply:
Because of `time = json['time']?.toInt() ?? 0;`
So `if(time == null) {xxx} ` will never run.
Try use `required this.time` and `int? tempTime` in function.

Reply:
right, my mistake. Should i make time `not required` and `time = json['time']?.toInt() ?? json['id']?.toInt()`. if both `json['time']` and `json['id']` is null, i will get `DateTime.now().milliseconds`

Reply:
I think `required time` can remind users.
`json['id']` is fake. I think this only works in the demo.In real use, the `json['time']` must be in effect.
Using `DateTime.now().milliseconds` may cause some problems( all data time will equal ).

Reply:
Error.I will fix it myself.